orchagent/intfsorch.cpp

```diff
@@ -265,9 +336,17 @@ void IntfsOrch::doTask(Consumer &consumer)
 
         vector<string> keys = tokenize(kfvKey(t), ':');
         string alias(keys[0]);
+
+        bool isSubIntf = false;
+        size_t found = alias.find(".");
+        if (found != string::npos)
+        {
+            isSubIntf = true;
+            SWSS_LOG_ERROR("sub interface: %s", alias.c_str());
```

Comment:
can you have a complete sentence for such error, "invalid sub interface", or something else. otherwise, how others know what error it is for this message?

Reply:
Removed. It was for debugging purpose. https://github.com/Azure/sonic-swss/pull/969#discussion_r320484206